packages/site/src/layout/ComponentView.tsx

```diff
@@ -187,44 +186,55 @@ export const ComponentView = () => {
   };
 
   return PageMeta ? (
-    <Grid>
-      <Grid.Cell size={isMinimal ? { xs: 12, md: 12 } : { xs: 12, md: 9 }}>
-        <Page width="fill" title={PageMeta.title}>
-          <PageWrapper>
-            <Box>
-              <Content spacing="large">
-                <Box direction="column" gap="small" alignItems="flex-end">
-                  <CodePreviewWindow>
-                    <AtlantisPreviewViewer />
-                  </CodePreviewWindow>
+    <div style={{ display: "flex", height: "100dvh" }}>
```

Comment:
We don't need Grid because there's only one item in here. I thought about hacking Grid with conditionally rendered empty cells to force the center alignment like `3 6 3`, but that seemed silly